usr/src/uts/common/fs/zfs/sys/zap.h

```diff
@@ -120,16 +111,6 @@ typedef enum zap_flags {
 
 /*
  * Create a new zapobj with no attributes and return its object number.
- * MT_EXACT will cause the zap object to only support MT_EXACT lookups,
- * otherwise any matchtype can be used for lookups.
- *
- * normflags specifies what normalization will be done.  values are:
- * 0: no normalization (legacy on-disk format, supports MT_EXACT matching
- *     only)
- * U8_TEXTPREP_TOLOWER: case normalization will be performed.
- *     MT_FIRST/MT_BEST matching will find entries that match without
- *     regard to case (eg. looking for "foo" can find an entry "Foo").
- * Eventually, other flags will permit unicode normalization as well.
```

Comment:
I agree that this comment is wrong and should be deleted.  Is there somewhere else that we document the `normflags`?  If not, could you add it to `zap_create_norm` in `zap_micro.c` or to `mz_normflags` / `zap_normflags` in zap_impl.h.?  I think the documentation should list what flags are acceptable, specifically one of `U8_TEXTPREP_NF*` plus `U8_TEXTPREP_TOUPPER`